### ISSUE TITLE
TypeError: map.options.ajaxquery.join is not a function

### ISSUE THREAD

**Issue by kghbln** [SemanticMediaWiki/SemanticMaps#93]
**Setup**

- MediaWiki 	1.27.0 (758cd9d) 22:24, 28. Jun. 2016
- PHP 	5.6.24-0+deb8u1 (apache2handler)
- MariaDB 	10.0.26-MariaDB-1~jessie
- Semantic Maps	3.4.0-alpha (354da01) 08:14, 20. Jun. 2016
- Maps	3.8.0 alpha (ce2971e) 08:23, 28. Jul. 2016

**Issue**
When using the "openlayers" and the "googlemaps" format on the [same page](http://sandbox.semantic-mediawiki.org/wiki/Issue/80_(SM)) "googlemaps" exits with "TypeError: map.options.ajaxquery.join is not a function": 
```
<anonym>
 load.php:2
_.B.trigger()
 js:96
ob()
 js:34
_.m.set()
 js:98
_.sc/<()
 js:48
c()
 map.js:57
jC.prototype.fitBounds()
 map.js:57
_.m.fitBounds/<()
 js:122
gc.prototype.Tb/</d[a]<()
 js:100
_.bc/<()
 js:44
gc.prototype.Tb/<()
 js:100
hc()
 js:46
gc.prototype.Tb()
 js:100
bound ()
 self-hosted:754
<anonym>
```
About three week ago when I was reporting https://github.com/SemanticMediaWiki/SemanticMaps/issues/80 this was still working so this is probably caused by a change after this date. If I [solely use "googlemaps" on a page](http://sandbox.semantic-mediawiki.org/wiki/Googlemaps) the issue cannot be observed.

**Comment by PeterTheOne** [comment]
Should be fixed by https://github.com/SemanticMediaWiki/SemanticMaps/pull/90, please check.

**Comment by kghbln** [comment]
> Should be fixed by #90, please check.

Affirmative. Thanks for the fast fix!
